AWS/DirectoryInsights/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to the AWS Directory Insights Serverless App will be documented in this file.
+
+## [1.2.0] - 2022-01-12
+
+### Added
+
+- Ability to filter available services instead of querying all directory insights data.
+  - Service can be set to query any of: "directory, radius, sso, systems, ldap, mdm" instead of the default "all" services
+
```

Comment:
Should we note that the cloud watch logs now post for every service? And that when no results are found they'll be noted as NoResults_[ServiceName] (including "all")?

Reply:
Yeah, I feel like that's probably good to include as well